Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,80 @@
-﻿using System;
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
-namespace BootCamp.Chapter
+namespace Homework
 {
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
-}
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			/*
+			 1.Print all the info based on the example message below:
+			   Tom Jefferson is 19 years old, 
+			   his weight is 50 kg and his height is 156.5 cm.
+			 */
+
+			double height = 156.5;
+			int weight = 50;
+			int age = 19;
+			string surName = "Jefferson";
+			string name = "Tom";
+
+			Console.WriteLine(name + " " + surName + " you are " + age + " years old, " +
```

Comment:
When making a string a small tip would be to add $ i front of the string that way you dont have to + all the time or use numbers like you did later on.
so: Console.WriteLine($"{name} {surName} you are {age} years old, your weight is {weight} Kg and your height is {height} cm")

Reply:
Thanks Max for the great Tip. Just changed those Lines. 
Mission Done!